apps/dash-clinical-analytics/app.py

```diff
@@ -0,0 +1,688 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+
+import numpy as np
+import pandas as pd
+import datetime
+from datetime import datetime as dt
+import pathlib
+
+app = dash.Dash(
+    __name__,
+    meta_tags=[{"name": "viewport", "content": "width=device-width, initial-scale=1"}],
+)
+
+server = app.server
+app.config.suppress_callback_exceptions = True
+
+# Path
+BASE_PATH = pathlib.Path(__file__).parent.resolve()
+
+DATA_PATH = BASE_PATH.joinpath("data").resolve()
+
+# Read data
+df = pd.read_csv(DATA_PATH.joinpath("clinical_analytics.csv"))
+
+clinic_list = df["Clinic Name"].unique()
+
+df["Admit Source"] = df["Admit Source"].fillna("Not Identified")
+admit_list = df["Admit Source"].unique().tolist()
+
+# Date
+# Format checkin Time
+df["Check In Time"] = df["Check In Time"].apply(
+    lambda x: dt.strptime(x, "%Y-%m-%d %I:%M:%S %p")
+)  # String -> Datetime
+
+# Insert weekday and hour of checkin time
+df["Days of Wk"] = df["Check In Hour"] = df["Check In Time"]
+df["Days of Wk"] = df["Days of Wk"].apply(
+    lambda x: dt.strftime(x, "%A")
+)  # Datetime -> weekday string
+
+df["Check In Hour"] = df["Check In Hour"].apply(
+    lambda x: dt.strftime(x, "%I %p")
+)  # Datetime -> int(hour) + AM/PM
+
+day_list = [
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
+]
+
+check_in_duration = df["Check In Time"].describe()
```

Comment:
```suggestion
check_in_duration = df["Check-In Time"].describe()
```

Didn't know about this `describe()` function -- super cool!